plugins/scaffolder/src/next/OngoingTask/OngoingTask.tsx

```diff
@@ -69,7 +72,7 @@ export const OngoingTask = (props: {
     [taskStream],
   );
 
-  const [logsVisible, setLogVisibleState] = useState(false);
+  const [logsVisible, setLogVisibleState] = useState(props.logs?.autoOpen);
```

Comment:
```suggestion
  const [logsVisible, setLogVisibleState] = useState(props.logs?.autoOpen ?? false);
```

to make it always boolean